internal/layerfile/layerfile_test.go

```diff
@@ -128,10 +127,10 @@ func TestToLayers_ValidateNameOfLayerDefinitions(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			_, err := tt.lf.ToLayers()
-			if err != nil {
-				assert.EqualError(t, tt.err, err.Error())
+			if tt.err == nil {
+				assert.NoError(t, err)
 			} else {
-				assert.NoError(t, tt.err)
+				assert.ErrorIs(t, err, tt.err)
 			}
```

Comment:
Here we should assert that there is an error or not based on `tt.err` begin defined or not. That's because what is defined in `tt` is what we expect to happen.

Reply:
i see, thanks for the pointer

Reply:
Thank you for the contribution! 🤘